app/recipe/tests/test_ingredients_api.py

```diff
@@ -0,0 +1,70 @@
+from django.urls import reverse
+from django.test import TestCase
+
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from core.models import Ingredient, Recipe
+
+from recipe.serializers import IngredientSerializer
+
+
+INGREDIENTS_URL = reverse('recipe:ingredient-list')
+
+
+class IngredientsApiTests(TestCase):
+    """Tests ingredients api"""
+
+    def setUp(self):
```

Comment:
If I'm right, you only need to instantiate `APIClient` once for the test clase. Then you could use `setUpTestData` instead, which is only called once at the beginning.

https://docs.djangoproject.com/en/3.1/topics/testing/tools/#django.test.TestCase.setUpTestData

Reply:
Thank you, will use this one instead then.